distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -65,3 +65,4 @@ export JAVA_MAX_MEM=4096M
 # export KARAF_BASE # Karaf base folder
 
 export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+export JAVA_OPTS=-"server -Xms128M -Xmx4096M -XX:+UnlockDiagnosticVMOptions -XX:+UnsyncloadClass -XX:+DisableAttachMechanism"
```

Comment:
Yes and it is up-to-date now. 
Updated the setenv script docs to account for the change to memory considerations here: 
https://github.com/codice/ddf/pull/2128/files#diff-b23b8e4895a7aae17f7b92e388b8a4beR13